### PR TITLE
Unit tests for models/repo_collaboration

### PR DESCRIPTION
**Pull request by ethantkoenig**
Unit tests for `models/repo_collaboration.go`

### CODE REVIEW

File: models/fixtures/access.yml [hunk 1]
```diff
@@ -6,6 +6,6 @@
 
 -
   id: 2
-  repo_id: 4
   user_id: 4
+  repo_id: 4
```
Comment (strk) [inline models/fixtures/access.yml:10]: please don't move things around for no reason :)

**Comment by strk** [comment]
LGTM, go tests go !

File: models/fixtures/access.yml (hunk 1 above)
Comment (ethantkoenig) [reply models/fixtures/access.yml:10]: Sorry, I wanted all of the access fixtures to have the same order of columns.

**Comment by lunny** [comment]
LGTM

**Pull request merged**